docs/courses/exercises/multiple-choice-exercises/responses.md

```diff
@@ -0,0 +1,68 @@
+# Writing `MultipleChoiceExercise` and `PureMultipleChoiceExercise` Responses
+
+This page provides guidance on writing good multiple choice responses.
```

Comment:
There is no mention of feedback messages, which are an excellent way to fix incorrect mental models a student may have.

Reply:
Thanks. Added via https://github.com/datacamp/authoring/commit/fd1c3e3192b86f6c32d239ccdf6482f8c50ff4fa